assets/js/googlesitekit/modules/datastore/__fixtures__/index.js

```diff
@@ -32,11 +29,10 @@ const alwaysActive = [ 'search-console', 'site-verification' ];
  * @param {...string} slugs Active module slugs.
  * @return {Object[]} Array of module objects.
  */
-export function withActive( ...slugs ) {
-	const activeSlugs = alwaysActive.concat( slugs );
+export function withActive( slugs ) {
```

Comment:
Could you please revert `withActive` and `withConnected` functions to use `...slugs` params because there are occurrences in the codebase where we rely on this approach.

Reply:
👍